civiclens/nlp/pipeline.py

```diff
@@ -76,7 +87,6 @@ def docs_have_titles():
             GROUP BY document_id HAVING COUNT(*) > 20;
             """
     if args.refresh:
```

Comment:
Could make sense to have this before `if last_updated is not None:` on line 77. That way if its a refresh you can skip running the longer query, not run that chunk just do the refresh query.

Reply:
nice, good catch!